components/layout/header.vue

```diff
@@ -116,11 +142,11 @@ import {
   XIcon,
 } from '@heroicons/vue/outline'
 import { ChevronDownIcon } from '@heroicons/vue/solid'
-
+import { Icon } from '@iconify/vue'
 const navigation = [
-  { name: 'Find Accomodation', to: '/guest' },
-  { name: 'UAH Exchange', to: '/exchange' },
-  { name: 'Recommend', to: '/add' },
+  { name: 'Add service', to: '/add' },
+  { name: 'Partnership', to: '/' },
```

Comment:
please create a new page for Partnership and Contact

Reply:
Okay

Reply:
Why did you resolve this conversation? The link is still not updated here.

Reply:
I've added it